desktop-widgets/filterwidget2.cpp

```diff
@@ -4,5 +4,5 @@ FilterWidget2::FilterWidget2(QWidget* parent)
 : QWidget(parent)
 , ui(new Ui::FilterWidget2())
 {
-    ui->setupUi(this);
+	ui->setupUi(this);
```

Comment:
when you submit a new series, it would be really nice to just not go through this. Sure, I could clean those up when applying, but rebasing and squashing those commits should be really easy for you.